core/src/main/java/com/linecorp/armeria/internal/common/ArmeriaHttpUtil.java

```diff
@@ -139,82 +139,82 @@ public boolean equals(AsciiString a, AsciiString b) {
     /**
      * The set of headers that should not be directly copied when converting headers from HTTP/1 to HTTP/2.
      */
-    private static final CharSequenceMap HTTP_TO_HTTP2_HEADER_BLACKLIST = new CharSequenceMap();
+    private static final CharSequenceMap HTTP_TO_HTTP2_HEADER_BLOCKLIST = new CharSequenceMap();
 
     /**
      * The set of headers that should not be directly copied when converting headers from HTTP/2 to HTTP/1.
      */
-    private static final CharSequenceMap HTTP2_TO_HTTP_HEADER_BLACKLIST = new CharSequenceMap();
+    private static final CharSequenceMap HTTP2_TO_HTTP_HEADER_BLOCKLIST = new CharSequenceMap();
 
     /**
      * The set of headers that must not be directly copied when converting trailers.
      */
-    private static final CharSequenceMap HTTP_TRAILER_BLACKLIST = new CharSequenceMap();
+    private static final CharSequenceMap HTTP_TRAILER_BLOCKLIST = new CharSequenceMap();
 
     static {
-        HTTP_TO_HTTP2_HEADER_BLACKLIST.add(HttpHeaderNames.CONNECTION, EMPTY_STRING);
-        HTTP_TO_HTTP2_HEADER_BLACKLIST.add(HEADER_NAME_KEEP_ALIVE, EMPTY_STRING);
-        HTTP_TO_HTTP2_HEADER_BLACKLIST.add(HEADER_NAME_PROXY_CONNECTION, EMPTY_STRING);
-        HTTP_TO_HTTP2_HEADER_BLACKLIST.add(HttpHeaderNames.TRANSFER_ENCODING, EMPTY_STRING);
-        HTTP_TO_HTTP2_HEADER_BLACKLIST.add(HttpHeaderNames.UPGRADE, EMPTY_STRING);
-        HTTP_TO_HTTP2_HEADER_BLACKLIST.add(ExtensionHeaderNames.STREAM_ID.text(), EMPTY_STRING);
-        HTTP_TO_HTTP2_HEADER_BLACKLIST.add(ExtensionHeaderNames.SCHEME.text(), EMPTY_STRING);
-        HTTP_TO_HTTP2_HEADER_BLACKLIST.add(ExtensionHeaderNames.PATH.text(), EMPTY_STRING);
+        HTTP_TO_HTTP2_HEADER_BLOCKLIST.add(HttpHeaderNames.CONNECTION, EMPTY_STRING);
```

Comment:
Somehow block list seems a bit different - blocking requests with these headers is what came to mind first. Perhaps `FILTERED_*` is a good pattern?

Reply:
It seems like a better idea. 😄

Reply:
`DISALLOWED_` also sounds good to me.